### ISSUE TITLE
MM Hunter spells ignore boss partial resists

### ISSUE THREAD

**Issue by mercred** [dalaranwow/dalaran-wow#5732]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently spells such as Chimera Shot, Chimera Shot:Serpent, Serpent Sting, Aimed Shot, Arcane Shot and Wild Quiver Auto Shot are not affected by partial resist. 

**Expected behaviour**: Tell us what should happen instead
According to the retail wol aforementioned spells should have been partially resisted. 

**Steps to reproduce the problem**:
Check any other hunter wol on the server and you will see the same.
1. Find any wol record from dalaran-wow 
2. Find any retail wol record at the time of 3.3.5
3. Compare hunter absorbs

**Include proofs for this behaviour**
Dala-wow hunters records on BQL: http://www.worldoflogs.com/reports/rt-ees4j3jq12cvlh8n/details/7/?s=8000&e=8273 ![No resist](https://user-images.githubusercontent.com/15626206/58750361-b26b1f80-8491-11e9-94d2-c7cf505f9f46.png)

Retail hunter records:
Retail hunter log taken march 2010 on BQL http://www.worldoflogs.com/reports/73nbh1fduzaa20oz/details/6/?s=7368&e=7633  
![hunter official](https://user-images.githubusercontent.com/15626206/58750358-a54e3080-8491-11e9-8783-4b6260f8d874.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mercred** [comment]
Any comment/label on this one? Seems like one of the major bugs that is easily fixable yet no update so far.

**Comment by mercred** [comment]
Bump. If implementing the exact formula is too complicated just add flat 6% resist against bosses and 4% for adds(following 2% average resist per level value).

**Comment by donniDevito** [comment]
Is it being worked on? Seems like one of the major bugs indeed.
